f8a_worker/models.py

```diff
@@ -362,13 +362,13 @@ class StackAnalysisRequest(Base):
     __tablename__ = "stack_analyses_request"
     id = Column(String(64), primary_key=True)
     submitTime = Column(DateTime, nullable=False)
-    requestJson = Column(JSON, nullable=False)
     origin = Column(String(64), nullable=True)
     result = Column(JSON, nullable=True)
     team = Column(String(64), nullable=True)
+    user_id = Column(String(64),nullable=False)
+    dep_snapshot = Column(JSONB, nullable=True)
```

Comment:
Follow the existing coding pattern, It seems it uses camelCase instead of snake_case.

Reply:
It looks like both type of column names are used. Ignore my comment.

Reply:
@arajkumar  dep_snapshot was already in snake case and also used in other repos where this is imported.DO you want me to change them all?

Reply:
Ignore this comment.